sql/rowexec/join_iters.go

```diff
@@ -463,7 +468,7 @@ func (i *fullJoinIter) Next(ctx *sql.Context) (sql.Row, error) {
 			}
 			if _, ok := i.seenLeft[key]; !ok {
 				// (left, null) only if we haven't matched left
```

Comment:
Is this comment still correct? It says `null` but you've replaced the `nil` in the LoC below.